magpie/api/management/user/user_utils.py

```diff
@@ -119,11 +121,21 @@ def _add_to_group(usr, grp):
 
     user_content = uf.format_user(new_user, new_user_groups)
 
+    # Create a token for the tmp_url, in case an error happens in the webhook services
+    token_id = uuid.uuid4()
+    webhook_token = models.TemporaryToken(
+        token=token_id,
+        operation=TokenOperation.WEBHOOK_ERROR.value,
+        user_id=new_user.id,
+        group_id=_get_group(group_name).id)
+    ax.evaluate_call(lambda: db_session.add(webhook_token), fallback=lambda: db_session.rollback(),
+                     http_error=HTTPForbidden, msg_on_fail=s.TemporaryToken_POST_ForbiddenResponseSchema.description)
+    tmp_url = webhook_token.url()
+
     # Force commit before sending the webhook requests, so that the user's status is editable if a webhook error occurs
     transaction.commit()
 
-    # Process any webhook requests
-    process_webhook_requests(WEBHOOK_CREATE_USER_ACTION, {"user_name": user_name, "tmp_url": "tmp_url:80/todo"}, True)
+    process_webhook_requests(WEBHOOK_CREATE_USER_ACTION, {"user_name": user_name, "tmp_url": tmp_url}, True)
```

Comment:
Can you add the comment typings in this function?
I think the function should receive a `WebhookAction` enum, and do `.value` inside it.

Reply:
I will change the WebhookAction list to use the ExtendedEnum (like the TokenOperation enum.), and will move the .value inside the function.